src/league_scraper_v1.py

```diff
@@ -0,0 +1,60 @@
+import pprint
+import re #regular expression
+import time
+from selenium import webdriver
+from bs4 import BeautifulSoup
+
+def scrapeMatchHistoryUrl(usernametext, passwordtext, matchUrl):
+
+	#options = webdriver.ChromeOptions()
+	#options.add_argument('--headless')
+	browser = webdriver.Chrome()
+	browser.maximize_window()
+	browser.get(matchUrl)
+	signinbutton = browser.find_element_by_css_selector(".riotbar-account-action")
+	signinbutton.click()
+
+
+	username = browser.find_element_by_name("username")
+	username.clear();
+	username.send_keys(usernametext)
+
+	password = browser.find_element_by_name("password")
+	password.clear();
+	password.send_keys(passwordtext)
+
+	submit = browser.find_element_by_css_selector(".mobile-button__submit")
+	submit.click()
+
+	time.sleep(8) #sleep while page loads
```

Comment:
there should be a better way to do this, but not important for now. can you leave a todo?

Reply:
slighter better version exists now